store/storetest/utils.go

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package storetest
+
+import (
+	"math/rand"
+	"strconv"
+	"time"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+)
+
+func init() {
+	rand.Seed(time.Now().UTC().UnixNano())
+}
+
+// NewTestId is used for testing as a replacement for model.NewId(). It is a [A-Z0-9] string 26
+// characters long. It replaces every odd character with a digit.
+func NewTestId() string {
+	newId := []byte(model.NewId())
+
+	for i := 1; i < len(newId); i = i + 2 {
+		s := strconv.Itoa(rand.Intn(10))
+		newId[i] = s[0]
+	}
```

Comment:
Do we have to randomize (and convert) further? We have everything we need in the original ID. What about:

```go
newId[i] = 48 + newId[i-1] % 10
```

This my inner [C](https://en.wikipedia.org/wiki/C_(programming_language)) heart suggesting :P

Reply:
No resistance 👍